ratelimit/redis.go

```diff
@@ -210,6 +218,12 @@ func setError(span opentracing.Span) {
 	}
 }
 
+func logError(span opentracing.Span, err error) {
+	if span != nil {
+		ext.LogError(span, err)
+	}
+}
```

Comment:
Do we want sampled span logging or sampled logging (`log.Errorf("whatever: v", err)`)?

Reply:
@szuecs

Reply:
I prefer sampled span logging because we have existing tracing support for redis.

Reply:
Why do you think sampled span logs bring value?
If these are sampled you likely won't find any of the sampled logs.

Reply:
@suzes I want to have visibility on what went wrong if redis call fails. Currently we only have light step traces to find if call failed.

Reply:
But then sampling makes no sense.

Reply:
I think the problem with sampling span logs is that it would be hard to find it - that is why OI proposed to use regular logs - those we'll see all at once.

Reply:
We've clarified with @nabeelbukhari that it is possible to filter error spans that also have logs.

Reply:
But then we need to log errors instead.

Reply:
I think it's more valuable to have errors in logs. If we see it's too many we can wrap it to stop too much logs. I think we should keep things simple and right now we create complexity for one narrow use.

Reply:
@szuecs @AlexanderYastrebov I changed logic and now we do logs instead of tagging to span. Please took another look.